### ISSUE TITLE
Event log has 9-byte EV_POST_CODE events that cause tpm2_eventlog to abort

### ISSUE THREAD

**Issue by osresearch** [tpm2-software/tpm2-tools#2017]
While trying to parse Lenovo/Phoenix event log, `tpm2_eventlog` aborts due to an `EV_POST_CODE` event that has 9 bytes, while the `parse_event2body()` function expects it will be as large as a 16-byte `UEFI_PLATFORM_FIRMWARE_BLOB`:

```
- Event[1]:
  PCRIndex: 0
  EventType: EV_EFI_PLATFORM_FIRMWARE_BLOB
  DigestCount: 2
  Digests:
    - AlgorithmId: sha1
      Digest: 379ecb760eee43b23c69c53fe631bb42d04b2047
    - AlgorithmId: sha256
      Digest: 05cae1f2b6f4a837b81283d08244e43db005f45468479de89a6198573df87632
  EventSize: 16
  Event:
    - BlobBase: 0xff970000
      BlobLength: 0x3c0000
- Event[1]:
  PCRIndex: 0
  EventType: EV_POST_CODE
  DigestCount: 2
  Digests:
    - AlgorithmId: sha1
      Digest: 01379597febfd617f8f1e27bdede6d3fc75fe97c
    - AlgorithmId: sha256
      Digest: bb9867bb2073d6fef711a2f5554bd53b48b95a7c3b9888277093660b21fc84e2
---- size 9 is insufficient for UEFI FW blob data (16)
ERROR: failed to parse tpm2 eventlog
ERROR: Unable to run ./tools/misc/tpm2_eventlog
```

~~Here is the [tpm.log](https://github.com/tpm2-software/tpm2-tools/files/4615945/tpm.log) from the Thinkpad X1 that causes the error.~~

**Comment by osresearch** [comment]
[tpm.log](https://github.com/tpm2-software/tpm2-tools/files/4620957/tpm.log)
This is actually the TPM log.

**Comment by osresearch** [comment]
Hex dumping the entry reveals the 9 bytes are `ACPI DATA`

**Comment by osresearch** [comment]
PR #2021 adds support for `EV_POST_CODE` events and displays them as byte strings.

**Comment by williamcroberts** [comment]
Fixed in #2021
